src/test/java/uk/gov/companieshouse/api/testdata/spec/CompanySpecTest.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.companieshouse.api.testdata.spec;
```

Comment:
Please use the same package as the class we are testing
```suggestion
package uk.gov.companieshouse.api.testdata.model.rest;
```

Reply:
renamed this.